tensorflow/lite/java/src/main/java/org/tensorflow/lite/Interpreter.java

```diff
@@ -102,7 +102,9 @@ public Options setUseNNAPI(boolean useNNAPI) {
      * Sets whether to allow float16 precision for FP32 calculation when possible. Defaults to false
      * (disallow).
      *
-     * <p>WARNING: This is an experimental API and subject to change.
+     * @deprecated Prefer using
+     * {@link org.tensorflow.lite.nnapi.NnApiDelegate.Options#setAllowFp16(boolean enable)}.
+     *
      */
     public Options setAllowFp16PrecisionForFp32(boolean allow) {
```

Comment:
You can add the literal @Deprecated annotation here, in code. See also setUseNNAPI and setNumThreads on the Interpreter class.

Reply:
Argh, I see. I added javadoc description but didn't add `@deprecated` to the method. Thanks